src/olympia/files/utils.py

```diff
@@ -210,11 +210,14 @@ def parse(self, minimal=False):
         if self.certinfo is not None:
             data.update(self.certinfo.parse())
 
+        data.update({
+                'name': self.find('name'),
+                'summary': self.find('description'),
```

Comment:
You can add it directly when building the `data` dict above, no need to do a `.update({})` call.

Reply:
https://github.com/mozilla/addons-server/pull/9461/commits/e1afc08754f3e68238d0918dde2f5b0f6b090b5c